libraries/src/Component/Router/RouterLegacy.php

```diff
@@ -17,6 +17,9 @@
  * Default routing class for missing or legacy component routers
  *
  * @since  3.3
+ * @deprecated  5.1 will be removed in 6.0
```

Comment:
```suggestion
 * @deprecated  5.1 will be removed in 7.0
```
It can't be removed before 7.0.